src/remoted/main.c

```diff
@@ -136,7 +136,7 @@ int main(int argc, char **argv)
 
     switch (is_worker){
         case -1:
-            merror("Cannot read cluster config %s", strerror(errno));
+            merror("Cluster node type is not valid [client/worker/master]. Cluster not configured. %s (%d)", strerror(errno), errno);
```

Comment:
The word client to refer to the worker is accepted by retrocompatibility. With this message seems that there are 3 node types.